File: PHPJS5/perhitungan.js
function printHeader() {
    console.log("---------------------");
    console.log("| Muhammad Naufal Athallah |");
    console.log("|      Bintaro      |");
    console.log("---------------------");
}

function calculate(operator, num1 = 0, num2 = 0) {
    switch (operator) {
        case "+":
            return num1 + num2;
        case "-":
            return num1 - num2;
        case "*":
            return num1 * num2;
        case "/":
            return num2 !== 0 ? (num1 / num2).toFixed(2) : "Tidak bisa dibagi nol!";
        case "%":
            return num2 !== 0 ? num1 % num2 : "Tidak bisa modulus dengan nol!";
        default:
            return "Operator tidak valid!";
    }
}

function main() {
    printHeader(); 

    while (true) {
        let operator = prompt("Masukkan Menu (+|-|/|*|%|stop):");
        if (operator.toLowerCase() === "stop") {
            console.log("Program Berhenti, Terima Kasih telah menggunakan program saya");
            break;
        }

        let num1 = parseFloat(prompt("Masukkan Nilai 1:"));
        let num2 = parseFloat(prompt("Masukkan Nilai 2:"));

        let result = calculate(operator, num1, num2);

        switch (operator) {
            case "+":
                console.log(`Hasil Penjumlahan ${num1} + ${num2} adalah ${result}`);
                break;
            case "-":
                console.log(`Hasil Pengurangan ${num1} - ${num2} adalah ${result}`);
                break;
            case "*":
                console.log(`Hasil Perkalian ${num1} * ${num2} adalah ${result}`);
                break;
            case "/":
                console.log(`Hasil Pembagian ${num1} / ${num2} adalah ${result}`);
                break;
            case "%":
                console.log(`Hasil Modulus ${num1} % ${num2} adalah ${result}`);
                break;
            default:
                console.log("Operator tidak valid! Silakan coba lagi.");
        }
    }
}

main(); //Menjalankan Program

File: PHPJS5/perhitungan.java
import java.util.Scanner;

public class perhitungan {
    
    public static void printHeader() {
        System.out.println("-----------------------------");
        System.out.println("| Muhammad Naufal Athallah |");
        System.out.println("|         Bintaro         |");
        System.out.println("-----------------------------");
    }

    public static double calculate(char operator, double num1, double num2) {
        switch (operator) {
            case '+':
                return num1 + num2;
            case '-':
                return num1 - num2;
            case '*':
                return num1 * num2;
            case '/':
                return (num2 != 0) ? num1 / num2 : Double.NaN; 
            case '%':
                return (num2 != 0) ? num1 % num2 : Double.NaN; 
            default:
                return Double.NaN; 
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        printHeader();

        while (true) {
            System.out.print("Masukkan Menu (+|-|/|*|%|stop): ");
            String input = scanner.next();

            if (input.equalsIgnoreCase("stop")) {
                System.out.println("Program Berhenti, Terima Kasih telah menggunakan program saya");
                break;
            }

            char operator = input.charAt(0);
            
            System.out.print("Masukkan Nilai 1: ");
            double num1 = scanner.nextDouble();
            
            System.out.print("Masukkan Nilai 2: ");
            double num2 = scanner.nextDouble();

            double result = calculate(operator, num1, num2);

            if (!Double.isNaN(result)) {
                System.out.printf("Hasil %s %.2f %c %.2f adalah %.2f%n", 
                                  (operator == '+' ? "Penjumlahan" :
                                   operator == '-' ? "Pengurangan" :
                                   operator == '*' ? "Perkalian" :
                                   operator == '/' ? "Pembagian" :
                                   "Modulus"),
                                  num1, operator, num2, result);
            } else {
                System.out.println("Operator tidak valid atau terjadi kesalahan!");
            }
        }

        scanner.close();
    }
}
